packages/contracts/src/contracts/current/protocol/Exchange/LibOrder.sol

```diff
@@ -33,7 +33,9 @@ contract LibOrder {
         "uint256 makerFeeAmount",
         "uint256 takerFeeAmount",
         "uint256 expirationTimeSeconds",
-        "uint256 salt"
+        "uint256 salt",
+        "bytes makerAssetProxyData",
```

Comment:
I prefer `makerAssetData` and `takerAssetData`. These fields could become more generic in the future, where some of the data ends up being used outside of a proxy.